tests/test_pois.py

```diff
@@ -0,0 +1,274 @@
+    'name': '',
+    'date': '01-01-00',
+    'description': '',
+    # 'map_year': ,
+    # 'x_coord': ,
+    'y_coord': 42,
+    'links': [],
+    'media': [],
+    'story_ids': []
+}
+
+map_year_invalid = 0
+story_id_invalid = 0
+poi_id_invalid = 0
+
+os.environ['POI_ID'] = '1' #initialize POI_ID as environment variable
+
+class POITests(unittest.TestCase):
+
+    def test0_1_create_poi(self):
+        r = requests.post('http://127.0.0.1:5000/pois', json=poi_create_complete)
+        response = r.json()
+        self.assertEqual(response['code'], 201)
+        self.assertEqual(response['result']['poi']['name'], poi_create_complete['name'])
+        # self.assertEqual(response['result']['poi']['date'], poi_edit['date'])
+        self.assertEqual(response['result']['poi']['description'], poi_create_complete['description'])
+        self.assertEqual(response['result']['poi']['map_year'], poi_create_complete['map_year'])
+        self.assertEqual(response['result']['poi']['x_coord'], poi_create_complete['x_coord'])
+        self.assertEqual(response['result']['poi']['y_coord'], poi_create_complete['y_coord'])
+        for i in range(len(response['result']['poi']['links'])):
+            self.assertEqual(response['result']['poi']['links'][i]['link_url'], poi_create_complete['links'][i]['link_url'])
+            self.assertEqual(response['result']['poi']['links'][i]['display_name'], poi_create_complete['links'][i]['display_name'])
+        for j in range(len(response['result']['poi']['media'])):
+            self.assertEqual(response['result']['poi']['media'][j]['content_url'], poi_create_complete['media'][j]['content_url'])
+            self.assertEqual(response['result']['poi']['media'][j]['caption'], poi_create_complete['media'][j]['caption'])
+        response_story_ids = [k['_id'] for k in response['result']['poi']['stories']]
+        self.assertEqual(sorted(response_story_ids), sorted(poi_create_complete['story_ids']))
+        os.environ['POI_ID'] = str(response['result']['poi']['_id'])
+
+    def test0_2_get_poi_by_id(self, poi_id=0):
+        if poi_id == 0:
+            poi_id = int(os.environ.get('POI_ID'))
+        r = requests.get('http://127.0.0.1:5000/pois/{}'.format(poi_id))
+        response = r.json()
+        self.assertEqual(response['code'], 200)
+        self.assertEqual(response['result']['poi']['_id'], poi_id)
+
+        self.assertEqual(response['result']['poi']['name'], poi_create_complete['name'])
+        # self.assertEqual(response['result']['poi']['date']
+        self.assertEqual(response['result']['poi']['description'], poi_create_complete['description'])
+        self.assertEqual(response['result']['poi']['map_year'], poi_create_complete['map_year'])
+        self.assertEqual(response['result']['poi']['x_coord'], poi_create_complete['x_coord'])
+        self.assertEqual(response['result']['poi']['y_coord'], poi_create_complete['y_coord'])
+
+        #Account for time zones (see above commented out line)
```

Comment:
explanations of comments should be the comment itself - you typically want your comments to be as close to the code it refers to as possible